src/LodeRunner.API.Test/IntegrationTests/ExecutingTestRun/TestRunExecution.cs

```diff
@@ -0,0 +1,283 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using LodeRunner.API.Models;
+using LodeRunner.API.Test.IntegrationTests.Controllers;
+using LodeRunner.API.Test.IntegrationTests.Extensions;
+using LodeRunner.Core.Models;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace LodeRunner.API.Test.IntegrationTests.ExecutingTestRun
+{
+    /// <summary>
+    /// Represents LoadTestConfigs.
+    /// </summary>
+    public class TestRunExecution : IClassFixture<ApiWebApplicationFactory<Startup>>
+    {
+        private const string TestRunsUri = "/api/TestRuns";
```

Comment:
What do you think about using SystemConstants.CategoryTestRunsPath?

Reply:
done